src/SFA.DAS.Commitments.Api.IntegrationTests/DatabaseSetup/DatabaseManagement.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlServer.Dac;
+using NUnit.Framework;
+using SFA.DAS.Commitments.Api.IntegrationTests.Tests;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.DatabaseSetup
+{
+    public class DatabaseManagement
+    {
+        private readonly string _connectionString;
+        private readonly string _databaseName;
+
+        public DatabaseManagement(string connectionString)
+        {
+            var builder = new SqlConnectionStringBuilder(connectionString);
+            _databaseName = builder.InitialCatalog;
+            builder.Remove("Initial Catalog");
+            _connectionString = builder.ConnectionString;
+        }
+
+        public async Task<bool> Exists()
+        {
+            using (var connection = new SqlConnection(_connectionString))
+            {
+                await connection.OpenAsync();
+                using (var command = new SqlCommand($"select name from sys.databases where name = '{_databaseName}'", connection))
+                {
+                    return (await command.ExecuteScalarAsync()) != null;
```

Comment:
brackets around the await aren't technically needed.

Reply:
Removed.